extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -126,7 +126,7 @@
 
     public static final String HIBERNATE_ORM_CONFIG_PREFIX = "quarkus.hibernate-orm.";
     public static final String NO_SQL_LOAD_SCRIPT_FILE = "no-file";
-    public static final String DEFAULT_PERSISTENCE_UNIT_NAME = "default";
+    public static final String DEFAULT_PERSISTENCE_UNIT_NAME = "<default>";
```

Comment:
What do you mean by "has special meaning", is it not possible for a user to speficy this name explicitly?

Reply:
The idea was to avoid a conflict between the default one and a potential named PU called "default". Thus why I added the special characters. Also, it's consistent with the Agroal extension.